### PR TITLE
Add script for migrating toilet IDs

### PR DESCRIPTION
**Pull request by rudigiesler**

### CODE REVIEW

**Comment by hodgestar** [comment]
:+1:

Could we perhaps follow this up with a quick PR to clean up the `ona_migration_script` folder? It no longer has only Ona migration scripts, and the scripts are all named `migrate_X` but don't say what they're changing. We should also try hook the tests up to Travis (although given that this repository has both Python and Javascript tests now, perhaps that's more of a nice to have).

File: ona_migration_script/migrate_toilet_codes.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+
+if __name__ == '__main__':
+    args = parser.parse_args()
+    session = create_session(args.username, args.password)
+    source_regex = re.compile(args.source_regex)
+    for toilet in get_all_toilets(session, args.url):
+        new_code = get_new_code(toilet, source_regex, args.target_regex)
+        if args.dryrun:
+            print '%s %s' % (toilet['code'], new_code)
```
Comment (hodgestar) [inline ona_migration_script/migrate_toilet_codes.py]: Last thought -- perhaps we should always print even if it's not a dry-run?

**Comment by rudigiesler** [comment]
Issue added: https://github.com/praekelt/go-imali-yethu-js/issues/42

So far there are only tests for this script, but we can add that to travis.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**